packages/beacon-node/src/chain/validation/aggregateAndProof.ts

```diff
@@ -142,17 +141,24 @@ async function validateAggregateAndProof(
   // -- i.e. get_ancestor(store, aggregate.data.beacon_block_root, compute_start_slot_at_epoch(store.finalized_checkpoint.epoch)) == store.finalized_checkpoint.root
   // > Altready check in `chain.forkChoice.hasBlock(attestation.data.beaconBlockRoot)`
 
-  const attHeadState = await getStateForAttestationVerification(
+  const shuffling = await getShufflingForAttestationVerification(
     chain,
-    attSlot,
     attEpoch,
     attHeadBlock,
-    RegenCaller.validateGossipAggregateAndProof
+    RegenCaller.validateGossipAttestation
   );
 
+  if (shuffling === null) {
+    throw new AttestationError(GossipAction.IGNORE, {
```

Comment:
This should never happen, the committee can always be computed. We just __choose__ to not compute it, which is very different. You should communicate with flow control the reason why the shuffle is not compute it and include it in the error payload.